x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/PlannerUtils.java

```diff
@@ -179,23 +180,25 @@ public static PhysicalPlan localPlan(
     }
 
     /**
-     * Extracts the ES query for the <code>@timestamp</code> field for the passed plan.
+     * Extracts a filter that can be used to skip unmatched shards on the coordinator.
      */
-    public static QueryBuilder requestTimestampFilter(PhysicalPlan plan) {
-        return detectFilter(plan, "@timestamp");
+    private static final Set<String> CAN_MATCH_FIELDS = Set.of("@timestamp", "event.ingested");
+
+    public static QueryBuilder canMatchFilter(PhysicalPlan plan) {
+        // metadata field like _index, _tier
+        return detectFilter(plan, f -> CAN_MATCH_FIELDS.contains(f) || f.startsWith("_"));
```

Comment:
this might pull in other fields starting with _ , such as _score or other user defined functions.